Services/EntitiesProvider.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Mrapps\BackendBundle\Services;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+use Mrapps\BackendBundle\Classes\Operator;
+
+class EntitiesProvider
+{
+    private $manager;
+
+    private $entities;
+
+    private $offset;
+    private $limit;
+    private $filters;
+    private $sorting;
+
+    private $queryParameters;
+
+    public function __construct(
+        EntityManager $manager
+    )
+    {
+        $this->manager = $manager;
+        $this->beginQuery();
+    }
+
+    public function beginQuery()
```

Comment:
Ho aggiunto questo metodo perchè altrimenti non c'era modo di ripulire i vari dati inseriti in una query precedente.